source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+  [HT802](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht802)
+  * includes a 5&nbsp;V power adapter and an Ethernet cable
+* RJ11 cable to connect the phone to the Grandstream
+
+## Setting up Grandstream
+
+1. Connect the RJ11 cable from the phone to the green socket on the Grandstream.
+   ![Grandstream HT801 ports](/images/assist/grandstream-ht801-interfaces.png)
+1. Connect the Grandstream to your network: 
+   * Plug the Ethernet cable into the blue socket and connect it to your router.
+1. Start up the Grandstream.
+   * Connect the power supply.
+   * Once the Grandstream has booted, the two LEDs for power and Ethernet light up solid blue. The phone LED won't light up.
+1. Log onto your router and identify the IP address of the Grandstream.
+1. Enter the IP address into a browser window and log onto the Grandstream *Device Configuration* software.
+   * The default credentials are:
+     * **Username**: `admin`
+     * **Password**: `admin`  
+   ![Login to Grandstream](/images/assist/grandstream_login.png)
+1. Open the **FXS Port** tab and in the **Offhook Auto-Dial** field, enter the Home Assistant IP address in the following format:
+   * \*47192\*168\*1\*100\*5060
+   ![Define IP for autodial](/images/assist/grandstream_autodial.png)
+   * Note: instead of 192\*168\*1\*100\*, enter the IP address of your Home Assistant instance.
+   * If you have a Grandstream HT802, there are two ports to configure. Repeat this step for the second port.
+   * On the bottom of the page, select **Apply**. 
+   ![Define IP for autodial](/images/assist/grandstream_apply.png)
+   * *Offhook Auto-Dial* means that when you pick up the phone, it directly calls Home Assistant. No need to dial.
+
+## Setting up the phone in Home Assistant
+
+1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & Services**" %} and select **Add integration**.
+1. Add the **Voice over IP** integration.   
+    ![Voice over IP integration](/images/assist/voip_install.png)
+1. Once you see the integration, pick up the phone.
+   * You should now hear the message *This is your smart home speaking. Your phone is connected, but you must configure it within Home Assistant.*
+   * The integration should now include a device and entities.
+    ![Voice over IP integration with device and entities](/images/assist/voip_device_available.png)
+   * Don't hear the voice? Try these [troubleshooting steps](projects/private-voice-assistant/voice_remote_classic_phone/#troubleshoot-grandstream).
+1. Allow calls.
+   * Calls from new devices are blocked by default since voice commands could be used to control sensitive devices, such as locks and garage doors.
+   * In the **Voice over IP** integration, select the **device** link.
+   * To allow this phone to control your smart home, under **Configuration**, enable **Allow calls**.   
+   ![Voice over IP integration - allow calls](/images/assist/voip_configuration.png) 
```

Comment:
I wonder if we should show a whole device info page screenshot, so it's clear where the configuration box is.